internal/demo/README.md

```diff
@@ -29,45 +29,73 @@ This guide will help you set up and run the Temporal Worker Controller locally u
      ```bash
      make create-cloud-mtls-secret
      ```
-   - Create a `skaffold.env` file:
+   - In `skaffold.example.env`, set:
```

Comment:
I think the purpose of `skaffold.example.env` is for people to copy it and edit their own copy, not to edit it directly. Not to be pedantic, but could you keep the demo instructions in line with that user flow?